client/scss/components/forms/_drop-zone.scss

```
@@ -0,0 +1,17 @@
+// file drop zones
+.drop-zone {
+  border-radius: 5px;
```

```suggestion
  border-radius: theme('borderRadius.DEFAULT');
```

Just wondering why we are not using the theme value here?

---

Upon further review, I’ll leave this as-is. Those are all existing styles and I don’t want the code to make it seem like it’s in line with our design system / tokens.